### PR TITLE
supporting info details page

### PR DESCRIPTION
**Pull request by lebeza**
image

Screen Shot 2019-06-05 at 3 40 46 PM

### CODE REVIEW

File: web/themes/custom/digital_toolkit/dist/css/style.css [hunk 1]
```diff
@@ -1090,6 +1090,37 @@ header {
...
+.page-node-type-supporting-information .detail-page {
+  margin-top: -56px;
+  position: relative;
+  z-index: -999; }
+  .page-node-type-supporting-information .detail-page .detail-page_summary {
+    padding-top: 80px;
+    margin-left: calc(50% - 50vw);
+    margin-right: calc(50% - 50vw);
+    padding-bottom: 60px;
+    background: #F3F3F3; }
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/dist/css/style.css]: change `#F3F3F3` to `$accent-light-grey`

File: web/themes/custom/digital_toolkit/dist/css/style.css [hunk 2]
```diff
@@ -1090,6 +1090,37 @@ header {
...
+
+.page-node-type-supporting-information .supporting-info_body .field--item:nth-child(even) {
+  /* padding-top: 66px; */
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/dist/css/style.css]: Please remove the comment.

File: web/themes/custom/digital_toolkit/dist/css/style.css [hunk 3]
```diff
@@ -1090,6 +1090,37 @@ header {
...
+  margin-left: calc(50% - 50vw);
+  margin-right: calc(50% - 50vw);
+  padding: 50px 0;
+  background: #F5f5f5; }
+
+.page-node-type-supporting-information .supporting-info_body .field--item:nth-child(odd) {
+  margin-left: calc(50% - 50vw);
+  margin-right: calc(50% - 50vw);
+  padding: 50px 0;
+  background-color: #fff; }
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/dist/css/style.css]: Please change `#fff` to `$body-bg`

**Comment by lebeza** [comment]
@shashank-naik the margin styles are converted to mixins now pushed for review
margin-left: calc(50% - 50vw); 
margin-right: calc(50% - 50vw);

**Pull request merged**